gradle/metalava-dependencies.gradle

```diff
@@ -6,18 +6,19 @@ dependencies {
     // Metalava isn't released yet. Check in its jar and explicitly track its transitive deps.
     metalava files('../metalava/metalava.jar')
 
-    metalava "com.android.tools.external.org-jetbrains:uast:27.2.0-alpha11"
-    metalava "com.android.tools.external.com-intellij:kotlin-compiler:27.2.0-alpha11"
-    metalava "com.android.tools.external.com-intellij:intellij-core:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-api:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-checks:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint-gradle:27.2.0-alpha11"
-    metalava "com.android.tools.lint:lint:27.2.0-alpha11"
-    metalava "com.android.tools:common:27.2.0-alpha11"
-    metalava "com.android.tools:sdk-common:27.2.0-alpha11"
-    metalava "com.android.tools:sdklib:27.2.0-alpha11"
-    metalava "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.20"
-    metalava "org.jetbrains.kotlin:kotlin-reflect:1.4.20"
+    metalava "com.android.tools.external.org-jetbrains:uast:30.3.0-alpha08"
+    metalava "com.android.tools.external.com-intellij:kotlin-compiler:30.3.0-alpha08"
+    metalava "com.android.tools.external.com-intellij:intellij-core:30.3.0-alpha08"
+    metalava "com.android.tools.lint:lint-api:30.3.0-alpha08"
```

Comment:
Why use alpha versions? 

https://mvnrepository.com/artifact/com.android.tools.lint/lint-api?repo=google
should we use [30.2.1](https://mvnrepository.com/artifact/com.android.tools.lint/lint-api/30.2.1) instead?

Reply:
This is auto-generated in order to update Metalava https://github.com/mapbox/mapbox-navigation-android/blob/main/docs/metalava.md#to-update-metalava and by the way Metalava it's not even stable 😬

Reply:
oh right, update is automatic through the build https://android.googlesource.com/platform/tools/metalava/ 

I'm not seeing anywhere there is a place to pull the stable channel 🤔 . but of course the main line is an alpha
https://android.googlesource.com/platform/tools/metalava/+/HEAD/DOWNLOADING.md

Reply:
Yeah, wondering if we should keep using a Google / Android "internal" tool i.e. docs are poor and not well maintained, changes in the repo are really hard to follow, as it's still in alpha they keep breaking SemVer which makes updating it very difficult sometimes...

We may want to consider something else 🤔

Also noting that the Maps team already integrates [Binary compatibility validator](https://github.com/Kotlin/binary-compatibility-validator) https://github.com/mapbox/mapbox-maps-android/pull/1120.